Composer/packages/client/src/components/CreationFlow/v2/CreateOptions.tsx

```diff
@@ -0,0 +1,268 @@
+import { DialogFooter } from 'office-ui-fabric-react/lib/Dialog';
+import { ScrollablePane, ScrollbarVisibility } from 'office-ui-fabric-react/lib/ScrollablePane';
+import { Selection } from 'office-ui-fabric-react/lib/DetailsList';
+import {
+  DetailsList,
+  DetailsListLayoutMode,
+  SelectionMode,
+  CheckboxVisibility,
+  DetailsRow,
+} from 'office-ui-fabric-react/lib/DetailsList';
+import { BotTemplate } from '@bfc/shared';
+import { DialogWrapper, DialogTypes } from '@bfc/ui-shared';
+import { NeutralColors } from '@uifabric/fluent-theme';
+import { RouteComponentProps } from '@reach/router';
+import { IPivotItemProps, Pivot, PivotItem } from 'office-ui-fabric-react/lib/Pivot';
+import { csharpFeedKey, nodeFeedKey } from '@botframework-composer/types';
+
+import { DialogCreationCopy, EmptyBotTemplateId, feedDictionary, QnABotTemplateId } from '../../../constants';
+import httpClient from '../../../utils/httpUtil';
+
+import { TemplateDetailView } from './TemplateDetailView';
+
+// -------------------- Styles -------------------- //
+
+const detailListContainer = css`
+  width: 50%;
+  height: 400px;
+  overflow: hidden;
+  float: left;
+  flex-grow: 1;
+`;
+
+const templateDetailContainer = css`
+  width: 50%;
+  height: 400px;
+  overflow: auto;
+  flex-grow: 1;
+  float: left;
+`;
+
+const pickerContainer = css`
+  position: relative;
+  height: 400px;
+  border: 1px solid #f3f2f1;
+`;
+
+const rowDetails = (disabled) => {
+  return {
+    root: {
+      color: disabled ? NeutralColors.gray80 : NeutralColors.black,
+      selectors: {
+        '&:hover': {
+          background: disabled ? NeutralColors.white : NeutralColors.gray30,
+          color: disabled ? NeutralColors.gray80 : NeutralColors.black,
+        },
+        '&.ms-DetailsRow.is-selected': {
+          background: disabled ? NeutralColors.white : NeutralColors.gray30,
+          color: disabled ? NeutralColors.gray80 : NeutralColors.black,
+        },
+      },
+    },
+  };
+};
+
+const rowTitle = (disabled) => {
+  return {
+    cellTitle: {
+      color: disabled ? NeutralColors.gray80 : NeutralColors.black,
+      selectors: {
+        ':hover': {
+          background: disabled ? NeutralColors.white : NeutralColors.gray30,
+          color: disabled ? NeutralColors.gray80 : NeutralColors.black,
+        },
+      },
+    },
+  };
+};
+
+const tableCell = css`
+  outline: none;
+  :focus {
+    outline: rgb(102, 102, 102) solid 1px;
+  }
+`;
+
+const content = css`
+  outline: none;
+`;
+
+const optionKeys = {
+  createFromScratch: 'createFromScratch',
+  createFromQnA: 'createFromQnA',
+  createFromTemplate: 'createFromTemplate',
+};
+
+// -------------------- CreateOptions -------------------- //
+type CreateOptionsProps = {
+  templates: BotTemplate[];
+  onDismiss: () => void;
+  onNext: (data: string) => void;
+  fetchTemplates: (feedUrls?: string[]) => Promise<void>;
+} & RouteComponentProps<{}>;
+
+export function CreateOptionsV2(props: CreateOptionsProps) {
```

Comment:
I don't like this V2 convention in the class name. Can you think of alternative ways to represent a different component path than hardcoding v2 in both the directories and the classes?

Reply:
Our plan is to remove the v2 convention when we make the hard shift to this creation experience prior to the build release.  That way this creation code path has the naming and file structure as the current path. 

Would there be an issue with that? I could change the 'v2' verbiage but pre build we will likely change it again to i.e. 'CreateOptions.ts' (once that file is no longer needed to support the current creation flow).